src/components/media-player/media-player-track-group.js

```diff
@@ -179,9 +182,9 @@ function MediaPlayerTrackGroup({
     const secondaryTrackDurations = trackStates.secondaryTracks.map(track => track.duration);
 
     if (!mainTrackDuration || (!!mainTrackDuration && secondaryTrackDurations.every(d => !!d))) {
-      onDuration(mainTrackDuration);
+      triggerDurationIfNeeded(mainTrackDuration);
     }
-  }, [trackStates, onDuration]);
+  }, [trackStates, triggerDurationIfNeeded]);
```

Comment:
This should fix it for `abc-notation` too then (Theresa created a bug https://educandu.atlassian.net/browse/EDU-882)

Reply:
Yes